s4sim/hardware/config_1shooter.py

```diff
@@ -297,13 +297,13 @@ def get_example():
     bnd["A"] = 0.09
     bnd["C"] = 0.87
     bands["ULFL1"] = bnd
-
+    
```

Comment:
Your editor leaves a lot of these orphan indentations. Is there an easy way for you to clean them up? You could try running `black` on the source.

Reply:
Weird, I will look into that/use a different editor in the future.